src/app/header/menu/menu.component.scss

```diff
@@ -32,40 +36,53 @@ ul {
 
   .menu-item {
     display: inline-block;
-    position: relative;
-    margin: 0 10px;
-    height: $menu-small-height;
     font-size: $font-size;
+    height: $menu-small-height;
+    margin: 0 10px;
+    position: relative;
     text-transform: uppercase;
 
     @include tablet {
       margin: 0;
     }
 
     a {
-      height: 100%;
-      display: flex;
       align-items: center;
+      display: flex;
+      height: 100%;
       padding: 0 10px;
+
+      &:focus {
```

Comment:
this link focus doesn't look so nice when hovered upon at the same time

![Focus hover](https://i.imgur.com/hidJPPw.png)

Reply:
I think this my be overkill style for inside the menu.